Cargo.toml

```diff
@@ -41,7 +41,7 @@ clap = { version = "4.0.29", features = ["derive"] }
 rcgen = { version = "0.13", default-features = false, features = ["pem"] }
 tokio = { version = "1.22.0", features = ["macros", "rt", "rt-multi-thread", "time"] }
 tracing = "0.1.37"
-tracing-subscriber = "0.3.16"
+tracing-subscriber =  { version = "0.3.16", features = ["env-filter"] }
```

Comment:
Nit: would prefer to squash this change into the commit that actually needs it.

Reply:
Done